.travis.yml

```diff
@@ -118,5 +118,6 @@ script:
 
   # compile and execute Python unit tests
   - pip install . --user
+  - PYTHONPATH="$PWD" python Test/Machine/test_machine.py
```

Comment:
This line is probably a leftover from testing?

Reply:
Right you are!